docker-compose.yml

```diff
@@ -16,4 +15,32 @@ services:
       interval: 5s
       retries: 5
     volumes:
-      - "./data:/refresh/data:z"
+      - "./data/gameserver:/refresh/data:z"
+  httpsproxy:
+    image: refresh-httpsproxy:latest
+    container_name: refresh-httpsproxy
+    build:
+      context: .
+      dockerfile: HttpsProxy.Dockerfile
+    restart: unless-stopped
+    ports:
+      - "10061:10061"
+    environment:
+      BUNKUM_DATA_FOLDER: /refresh/data
+    volumes:
+      - "./data/httpsproxy:/refresh/data:z"
+    profiles: ["optional-services"]
+  presenceserver:
+    image: refresh-presenceserver:latest
+    container_name: refresh-presenceserver
+    build:
+      context: .
+      dockerfile: PresenceServer.Dockerfile
+    restart: unless-stopped
+    ports:
+      - "10072:10072"
```

Comment:
PresenceServer opens two ports, `10072` (the presence server port) and `10061` (the Refresh communication HTTP port)